subprojects/ide-provisioning/src/main/java/org/gradle/profiler/ide/Ide.java

```diff
@@ -0,0 +1,9 @@
+package org.gradle.profiler.ide;
+
+import org.jetbrains.annotations.NotNull;
+
+public interface Ide {
+    @NotNull
+    String getVersion();
```

Comment:
I'm doubting what data to use for providing. From one perspective `version` is a good and simple declaration of which release of IDE we want to get. However, there is a `buildNumber` as well, and it's possible to have a IDE with same version with different buildNumbers(I thinks it's the case for nightlies)

Reply:
We can always change that later, if we need a need to. So I think version is fine for now